dgm/01_grass_import_openNRW_dgm1_epsg25832.sh

```diff
@@ -116,13 +116,16 @@ for zip in $(ls dgm*_EPSG25832_XYZ.zip) ; do
         cat $dgm | tr -s ' ' ' ' | r.in.xyz input=- separator=space method=mean output=$name
         # shift from EPSG:4647 to EPSG:25832, adjusting false easting
         r.region map=$name e=e-32000000 w=w-32000000
+        # shift raster for half pixel because in the XYZ file the left lower corner is given
+        g.region n=n+0.5 s=s-0.5 w=w-0.5 e=e+0.5 -p
```

Comment:
```suggestion
        g.region map=$name 
        g.region n=n+0.5 s=s+0.5 w=w+0.5 e=e+0.5 -p
```
for a shift of the whole map to the east and north, half a pixel must be **added** to all extents